### ISSUE TITLE
Command completion

### ISSUE THREAD

**Issue by edsu** [DocNow/twarc#444]
It would be nice if command completion worked with twarc2 since [click supports it](https://click.palletsprojects.com/en/7.x/bashcomplete/). I 'm not sure why it isn't working already since I thought it was on by default.

**Comment by kieranjol** [comment]
That would be really cool. I always thought that this was the kind of thing
that had to be enabled at the shell level or something.

**Comment by igorbrigadir** [comment]
Yeah it "works" already, but it doesn't do this step: https://click.palletsprojects.com/en/7.x/bashcomplete/#activation

I got it to work after executing 

`eval "$(_TWARC2_COMPLETE=source_bash twarc2)"`

(but this is shell specific so it's gonna be different for others)
